SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsImpl.cc

```diff
@@ -727,3 +748,13 @@ double QuickTrackAssociatorByHitsImpl::weightedNumberOfTrackHits(const Trajector
   }
   return sum;
 }
+
+const TrackingRecHit* QuickTrackAssociatorByHitsImpl::getMostProbableHit(const SiTrackerMultiRecHit*& mHit) const{
```

Comment:
There is no need to pass pointer by reference, i.e. `const SiTrackerMultiRecHit *mHit` is enough.